Source/SwiftLintFramework/Rules/RuleConfigurations/TrailingWhitespaceConfiguration.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  TrailingWhitespaceConfiguration.swift
+//  SwiftLint
+//
+//  Created by Reimar Twelker on 12.04.16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+
+public struct TrailingWhitespaceConfiguration: RuleConfiguration, Equatable {
+    var severityConfiguration = SeverityConfiguration(.Warning)
+    var ignoresEmptyLines = false
+
+    public var consoleDescription: String {
+        return severityConfiguration.consoleDescription +
+            ", ignores_empty_lines: \(ignoresEmptyLines ? "true" : "false")"
+    }
+
+    public init(ignoresEmptyLines: Bool) {
+        self.ignoresEmptyLines = ignoresEmptyLines
+    }
+
+    public mutating func applyConfiguration(configuration: AnyObject) throws {
+        guard let configuration = configuration as? [String: AnyObject] else {
+            throw ConfigurationError.UnknownConfiguration
+        }
+
+        if let ignoresEmptyLinesString = configuration["ignores_empty_lines"] as? Bool {
```

Comment:
rather than use `if let` here, please reduce this to one line:

```swift
ignoresEmptyLines = (configuration["ignores_empty_lines"] as? Bool == true)
```